File: ExampleProject.UI/Core/ExampleProject/ServerConfig.ts
import { ServerConfigBase } from "codeshell/core";

export class ServerConfig implements ServerConfigBase {
    Locale: string = "ar";
    Domain: string = "ExampleProject";

    static _instance?: ServerConfig;
    static get Instance(): ServerConfig {
        if (!this._instance) {
            this._instance = new ServerConfig();
        }
        return this._instance;
    }
}

File: ExampleProject.UI/Models/ServerConfig.cs
using CodeShellCore.Web.Moldster;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ExampleProject.UI.Models
{
    public class ServerConfig : IServerConfig
    {
        public string BaseURL { get; set; }
        public string Domain { get; set; }
        public string Locale { get; set; }
        public string Env { get; set; }
        public string Version { get; set; }
        public string Hash { get; set; }
    }
}
